Radio/index.jsx

```diff
@@ -67,19 +84,42 @@ export default React.createClass({
     } = this.props
 
     const classNames = classNamesBind.bind({ ...defaultStyles, ...styles })
+    const singleOption = options.length === 1
+    const baseStyle = customize ? { borderRadius: customize.borderRadius } : undefined
+    const labelStyle = customize ? { color: customize.textPrimaryColor } : undefined
+    const descriptionStyle = customize ? { color: customize.textSecondaryColor } : undefined
+    const bulletStyle = customize
+      ? {
+        backgroundColor: customize.backgroundColor,
+        borderColor: customize.backgroundColor
+      }
+      : undefined
+    const checkmarkStyle = customize ? { backgroundColor: customize.bulletColor } : undefined
 
     return (
       <div
         className={classNames(baseClass, {
-          borderful,
           borderless,
-          'is-disabled': disabled,
           'is-focused': focus != null
         }, className)}
+        style={baseStyle}
         {...remainingProps}>
-        {
-          options.map(({key, label, description}) => [
-            <input
+        {options.map((option) => {
+          const {
+            key,
+            label,
+            description,
+            disabled: toExtractDisabled, // eslint-disable-line no-unused-vars
+            aside,
+            content,
+            leftPad,
+            ...restOfProps
+          } = option
+
+          const disabled = allDisabled || option.disabled
```

Comment:
Why are you destructing `disabled` to `toExtractDisabled` if you still refer to it as `option.disabled` here?